lib/resources/oidc.js

```diff
@@ -102,15 +103,19 @@ module.exports = (service, endpoint) => {
       const client = await getClient();
       const code_verifier = generators.codeVerifier(); // eslint-disable-line camelcase
 
+      // State is not strictly required because PKCS (code_challenge) protects us from CSRF attacks.
```

Comment:
It feels important to then mention why it's there! Maybe just add "but some IDPs always send it"

Reply:
done